app/assets/stylesheets/components/user.scss

```diff
@@ -442,21 +441,166 @@ main {
+              }
+            }
+            &:nth-child(5n) {
+              border-top-right-radius: 6px;
+              border-bottom-right-radius: 6px;
+              a {
+                border-right: 0;
+              }
+            }
+            &:nth-child(5n + 1) {
+              border-top-left-radius: 6px;
+              border-bottom-left-radius: 6px;
+            }
+          }
+          @media #{$one-column} {
+            width: calc(100% / 3);
+            border-radius: 0;
+            &:nth-child(6n) {
+              border-radius: 0;
+              a {
+                border-right: 2px solid #999;
+              }
+            }
+            &:nth-child(6n + 1) {
+              border-radius: 0;
+              a {
+                border-right: 2px solid #999;
+              }
+            }            
+            &:nth-child(4n) {
+              border-radius: 0;
+              a {
+                border-right: 2px solid #999;
+              }
+            }
+            &:nth-child(4n + 1) {
+              border-radius: 0;
+              a {
+                border-right: 2px solid #999;
+              }
+            }
+            &:nth-child(3n) {
+              border-top-right-radius: 6px;
+              border-bottom-right-radius: 6px;
+              a {
+                border-right: 0;
+              }
+            }
+            &:nth-child(3n + 1) {
+              border-top-left-radius: 6px;
+              border-bottom-left-radius: 6px;
+            }
+          }
           &:last-child {
-            border-right: 0;
+            border-top-right-radius: 6px;
+            border-bottom-right-radius: 6px;
+            a {
+              border-right: 0;
+            }
+          }
+          @media #{$mobile} {
+            width: calc(100% / 2);
+            border-radius: 0!important;
+            margin-bottom: 0px;
+            border-bottom: 1px solid #000;
+            a {
+              opacity: 1!important;
+            }
+            &:nth-child(6n) {
+              border-radius: 0;
+              a {
+                border-right: 2px solid #000;
+              }
+            }
+            &:nth-child(6n + 1) {
+              border-radius: 0;
+              a {
+                border-right: 2px solid #000;
+              }
+            }            
+            &:nth-child(4n) {
+              border-radius: 0;
+              a {
+                border-right: 2px solid #000;
+              }
+            }
+            &:nth-child(4n + 1) {
+              border-radius: 0;
+              a {
+                border-right: 2px solid #000;
+              }
+            }
+            &:nth-child(3n) {
+              border-radius: 0;
+              a {
+                border-right: 2px solid #000;
+              }
+            }
+            &:nth-child(3n + 1) {
+              border-radius: 0;
+              a {
+                border-right: 2px solid #000;
+              }
+            }
```

Comment:
Is there a reason these are expanded or can we collapse them to 

&:nth-child(6n), 
&:nth-child(6n + 1),
etc